core/src/bitcoin/bech32/Bech32Factory.cpp

```diff
@@ -35,7 +35,8 @@
 namespace ledger {
     namespace core {
         Option<std::shared_ptr<Bech32>> Bech32Factory::newBech32Instance(const std::string &networkIdentifier) {
-            if (networkIdentifier == "btc" || networkIdentifier == "btc_testnet") {
+            auto btcBech32Identifiers = std::vector<std::string>{"btc", "btc_testnet", "dgb", "ltc"};
+            if (std::find(btcBech32Identifiers.begin(), btcBech32Identifiers.end(), networkIdentifier) != btcBech32Identifiers.end()) {
```

Comment:
Please add `const` qualifier.

I have a other suggestion here it's to replace `std::vector` by `std::string[]` then use `std::cbegin(btcBech32Identifiers)` and `std::cend(btcBech32Identifiers`. 
With that modification we remove the runtime overhead to construct the `std::vector`.

Reply:
ACK for the const qualifier, for the second part this is definitely to be part of the enhancements your working it, because we have many places where we could have to same optim (better to isolate in another PR)